onnx/defs/data_type_utils.cc

```diff
@@ -172,7 +173,7 @@ std::string DataTypeUtils::ToDataTypeString(int32_t tensor_data_type) {
   TypesWrapper& t = TypesWrapper::GetTypesWrapper();
   auto iter = t.TensorDataTypeToTypeStr().find(tensor_data_type);
   if (t.TensorDataTypeToTypeStr().end() == iter) {
-    throw std::invalid_argument("Invalid tensor data type.");
+    return "Invalid tensor data type.";
```

Comment:
The 1st iter of the previous PR #2888 was trying to return an empty string, does the "Invalid tensor data type." actually mean anything to the caller?

Reply:
Agree. I see pros/cons, but for code that handles this, comparing against an empty string is simpler and easier.

Reply:
Or, a named string constant.

Reply:
Resolved with an empty string.